app/src/main/java/ca/bc/gov/bchealth/utils/PhnHelper.kt

```diff
@@ -0,0 +1,40 @@
+package ca.bc.gov.bchealth.utils
+
+import android.content.Context
+import androidx.core.widget.doOnTextChanged
+import ca.bc.gov.bchealth.R
+import com.google.android.material.textfield.TextInputLayout
+
+/*
+* Created by amit_metri on 10,February,2022
+*/
+class PhnHelper {
+
+    fun validatePhnData(
+        textInputLayout: TextInputLayout,
```

Comment:
Api should not take view reference, you can pass a string and then validate it, and based on validation you just need to display error state based on output